cli/listener/src/main.rs

```diff
@@ -27,8 +27,12 @@ use std::str::FromStr;
 use tiny_http::{Header, Request, Response, ResponseBox, Server};
 use url::{form_urlencoded, Url};
 
+// Name put in the solido table `pool`.
 const SOLIDO_ID: &str = "solido";
 
+// Offset from the first Epoch's entry to use as data point.
```

Comment:
Ideally, we would offset it from the start of the epoch, not from the first entry. There is probably some code in `solana_program::clock` that we can use to compute it. What do you think?

**Edit**: Also, it wasn’t clear to me that this was about rows, not an offset in slots. We shouldn’t do it on rows because the poll interval is configurable.